### PR TITLE
[stable10] Use '.part' name appended to the file name for encryptall

### PR DESCRIPTION
**Pull request by sharidas**
When dealing with copy and rename operation in
encryptall, its better to use '.part' appended
to the file name.

Signed-off-by: Sujith H <sharidasan@owncloud.com>

## Description

While handling encrypt operation, its better to use '.part' file name during copy and rename.
Also no encryption should be made done for incoming shares.

## Motivation and Context

While handling encrypt operation, its better to use '.part' file name during copy and rename.
Also no encryption should be made done for incoming shares.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sharidas** [comment]
Original PR: https://github.com/owncloud/encryption/pull/35

**Review by PVince81 — APPROVED** [review]
:+1:

File: tests/drone/install-server.sh [hunk 1]
```diff
@@ -144,4 +144,4 @@ php occ app:enable files_trashbin
 php occ app:enable files_versions
 php occ app:enable provisioning_api
 php occ app:enable federation
-php occ app:enable federatedfilesharing
+php occ app:enable federatedfilesharing
```
Comment (patrickjahns) [inline tests/drone/install-server.sh]: this change is not relevant to the PR - why is it there?

**Comment by PVince81** [comment]
hmm strange, once again it says that `getTimestamp()` is not called.

can you remove it ? or if you want to mock `time()` then use the `TimeFactory` and mock that

**Pull request merged**